server/db/models/room.js

```diff
@@ -1,13 +1,49 @@
 const { Sequelize } = require('sequelize');
 const db = require('../db');
 
+const Team = require('./team');
+
 const Room = db.define('room', {
```

Comment:
I like how this room is set up, can really see how teams will be connected to rooms and players